Formula/g/gnmic.rb

```diff
@@ -0,0 +1,28 @@
+class Gnmic < Formula
+  desc "GNMI CLI client and collector"
+  homepage "https://gnmic.openconfig.net"
+  url "https://github.com/openconfig/gnmic.git",
+      tag:      "v0.36.2",
+      revision: "a7844a6d7d47c302df1fabb845dd22e4e4b70e22"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
Aren't these the default values?

Reply:
I'm not sure, I followed https://github.com/Homebrew/homebrew-core/blob/master/Formula/x/xq.rb for the format as I didn't see any docs especially for go formulas. I can remove that if it is not needed.

Reply:
I have removed the environment variables and the Formula builds fine without them.